.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
:eyes: what's this about? Does it work on vim?

Also why do we need it here? Isn't the CI style check enough?